src/client.ts

```diff
@@ -129,16 +128,12 @@ const client = ({
       return projectsService;
     },
     modifyFile: () => {
-      if (modifyFile) return modifyFile;
-
-      modifyFile = new ModifyFile({
+      return new ModifyFile({
```

Comment:
This is interesting. I wonder if we need to rethink singletons for the other services too.

Reply:
Do you know of any side effects that might pop up going this route?

Reply:
I think the other services are alright as singletons because they have individual methods that create streams. `ModifyFile` is unique in that it creates the stream when it is constructed. The one side effect that comes to mind is that there's the potential to open an overwhelming amount of streams if you have a lot of `ModifyFile` calls concurrently.